src/foam/nanos/app/ContextLookupAppConfigService.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.app',
+  name: 'ContextLookupAppConfigService',
+  implements: [
+    'foam.nanos.app.AppConfigService'
+  ],
+  methods: [
+    {
+      name: 'getAppConfig',
+      type: 'foam.nanos.app.AppConfig',
+      javaCode: `
```

Comment:
Have this method take a context as the first argument and then look the AppConfig up from there. This is better because we customize the appConfig in the user's session with their root URL.

Reply:
Done